test/CommandTest.php

```diff
@@ -332,9 +351,9 @@ public function testMessageOnEnableWhenModuleIsAlreadyEnabled($type)
         $this->setUpModule($modulesDir, 'App', $type);
 
         $this->console
-            ->writeLine(Argument::containingString('Autoloading rules already exist for the module "App"'))
-            ->shouldBeCalled();
-        $this->assertNotHelpOutput(STDERR);
+            ->expects($this->once())
+            ->method('writeLine')
+            ->with('Autoloading rules already exist for the module "App"');
```

Comment:
This and other methods where I remove the call to either `assertNotHelpOutput()` or `assertHelpOutput()` were cases where I hit a serialization bug in PHPUnit that exists due to how mock object constraints are created. Most of these have a closure memoized internally, and these cannot be serialized, which raises an error. The only way I could get tests to run for these was to:

- Assert an _exact_ number of invocations.
- Use _exact_ expectations for arguments (vs using a constraint method like `stringContains()`).

Since I'm now asserting that `writeLine()` is called an exact number of times, I know that the help output is NOT being created, allowing me to remove that assertion (and that assertion, because it essentially required a `stringContains()` comparison to be readable, was causing the seriailzation error).